OpenRA.Game/Primitives/int2.cs

```diff
@@ -36,9 +35,19 @@ public struct int2
 		public int2 Sign() { return new int2(Math.Sign(X), Math.Sign(Y)); }
 		public int2 Abs() { return new int2(Math.Abs(X), Math.Abs(Y)); }
 		public int LengthSquared { get { return X * X + Y * Y; } }
-		public int Length { get { return Exts.ISqrt(LengthSquared); } }
+		public int Length { get { return (int)Math.Sqrt(LengthSquared); } }
```

Comment:
IIRC Sqrt in the past has had imprecision issues between each of our platforms, which is why we have ISqrt.